manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -110,8 +110,18 @@ spec:
     rules:
     - alert: UsingDeprecatedAPIExtensionsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated extensions/v1beta1 API that will be removed soon.
+        summary: "A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version"
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="extensions",version="v1beta1",resource!~"ingresses|",client!~"hyperkube/.*|cluster-policy-controller/.*"}
+        increase(
+          apiserver_request_count{
```

Comment:
Please note that `apiserver_request_count` is a deprecated metric and we are dropping those metrics from openshift as well. We should be using `apiserver_request_total` instead.

Reply:
@LiliC drop when? This PR is meant for 4.3 most importantly. If you're dropping this in 4.4 - fine change, if in the next we'll fix it then.

Reply:
Yes we are dropping this metric start with 4.4. It has been deprecated since 1.14 k8s version and has a replacement which is `apiserver_request_total ` so just changing it to that should just work ™️ .

Reply:
Made this change, and I verified the alert still fires as expected in a test cluster.

Reply:
:+1: @LiliC does the new metric will be working fine in 4.3 as well? Just ensuring before we switch it in the backport too, or the backport has to be different?

Reply:
My test cluster is running 4.3.0 GA, so I can confirm that this works.